app/posts/apply-for-teacher-training/2023-03-24-merging-the-personal-statement.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Merging the personal statement 
+description: We merged the two personal statement sections into one and improved the guidance around what we say to candidates to help them write a good personal statement
+date: 2023-03-24
+tags:
+  - personal statement
+#ogImage:
+#src: /apply-for-teacher-training/merged-personal-statement.png
+#alt: "Screenshot of the personal statement page, with guidance text and a text box."
+
+---
+
+Previously, the personal statement had two sections on the application form: ‘Why you want to teach’ (600 words) and ‘Your suitability to teach a subject or age group’ (400 words). 
+
+We merged these personal statement sections into one, and improved the guidance around what we say to candidates to help them write a good personal statement. 
+
+We tested this design and improved the content shown to candidates after conducting research. We released this feature, via a feature flag, on 20th March 2023.
```

Comment:
Let’s put in the date it actually goes live (once we've confirmed it)?

```suggestion
We tested this design and improved the content shown to candidates after conducting research. We released this feature to candidates on 3 April 2023.
```

Reply:
So remove line for the time being @frankieroberto ?

Reply:
Yeah, then add it back when we launch? Or say "We plan to..." and we'll update it post-launch?